### DISCUSSION TITLE
Can i import ViewInspector for inspect the element in run time?

### DISCUSSION THREAD

**Discussion by AsafInbar** [nalexn/ViewInspector#175] [Q&A]
Hi i am trying to create Analytics application and i will need the ability to scan the SwiftUI view hierarchy in real time
when i tried to import the ViewInspector i get some error regards the XCTest framework
so my question is there a way to include the framework inside an App and inspect the view in real time?

**Comment by nalexn** [comment]
Hey, the library is [not designed to be used in the main build target](https://github.com/nalexn/ViewInspector#how-do-i-add-it-to-my-xcode-project).
You can try forking it and getting rid of a few files that import `XCTest.framework`

**Reply by AsafInbar** [reply]
Thanks for your answer 
i will try and let you know

**Reply by alsidor** [reply]
Hi @AsafInbar ! Investigating the similar task. Any success on running inside the main target?

**Reply by nalexn** [reply]
You guys can fork the repo and simply remove a few helpers that are built for XCTest, it's quite easy to decouple and make the project main target friendly.
